.gitignore

```diff
@@ -2,7 +2,6 @@ _build/
 _site/
 Properties/
 node_modules/
-wwwroot/
```

Comment:
We'd like to keep the default of ignoring wwwroot then overriding in the few cases you need that directory